eRecruiter.Api/Responses/Application/ApplicationReferrerResponse.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+
+namespace eRecruiter.Api.Responses
+{
+    /// <summary>
+    /// Response for application queries on referrer.
+    /// </summary>
+    public class ApplicationReferrerResponse
```

Comment:
A fact that should be fixed, but will add the ``JobId`` anyway as it make sense.